core/shared/src/main/scala/org/http4s/Entity.scala

```diff
@@ -18,10 +18,17 @@ package org.http4s
 
 import cats.Monoid
 import cats.syntax.all._
+import cats.~>
 
 final case class Entity[+F[_]](body: EntityBody[F], length: Option[Long] = None)
 
 object Entity {
+  implicit class InvariantOps[F[_]](private val self: Entity[F]) extends AnyVal {
```

Comment:
Nitpick: implciit conversion via an implicit class vs an implicit method creating an instance of that class? If I had to guess, it's probably easier to control the binary evolution of the latter. But, maybe it doesn't make a difference at all actually 🤔

Reply:
Actually, when I started on the next slice I changed this to an instance method with an extra type parameter, so I'll just make that change now instead.